app/views/articles/_youtube.html.erb

```diff
@@ -1,4 +1,10 @@
-<div class="youtube"></div>
+<div class="youtube-container">
+  <h3>Related Videos</h3>
+  <p>The videos below are based on the following tags: <%= @article.cached_tag_list %>.</p>
```

Comment:
@dreardon1021 Just added a short description; feel free to revise and change the test.

Reply:
@dreardon1021 Can you take a look at this and make sure the styling is aesthetically pleasing?